### PR TITLE
[devtools-translate] Korean - What's New in DevTools (Chrome 108)

### LINKED ISSUE #4128: [devtools-translate] Korean - What's New in DevTools (Chrome 108)
Post: https://developer.chrome.com/blog/new-in-devtools-108/
Translated by: Nov-11

Translator: @TORU0239
Reviewers: @techhtml, @cwdoh

### PR DESCRIPTION
**Pull request by TORU0239**
Fixes #4128

### CODE REVIEW

**Comment by TORU0239** [comment]
@techhtml @cwdoh Please review. :) 
cc. @jecfish

File: site/ko/blog/new-in-devtools-108/index.md [hunk 1]
```diff
@@ -26,38 +27,41 @@ draft: true
...
 
 
 <!-- ## Step through comma-separated expressions {: #debugging } -->
-
+## 쉼표로 구분된 표현을 통한 {: #debugging }
 <!-- You can now step through comma-separated expressions during debugging. This improves the debuggability of minified code. -->
-
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/4lUgUfPMhD9qxtZ7uvHV.png", alt="Step through comma-separated expressions.", width="800", height="473" %}
+이제 여러분은 디버깅 중에 쉼표로 구분된 표현식을 통해 다음 행으로 건너갈 수 있습니다. 이는 최소화된 코드의 디버깅 가능성을 개선할 수 있습니다.
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/4lUgUfPMhD9qxtZ7uvHV.png", alt="쉼표로 구분된 표현을 통해 다음 행으로 진행", width="800", height="473" %}
```
Comment (techhtml) [inline site/ko/blog/new-in-devtools-108/index.md:59]: Title, TOC is different translation.
How about using same translation?

`쉼표로 구분된 표현식을 통해 다음 행으로 진행`

Comment (TORU0239) [reply site/ko/blog/new-in-devtools-108/index.md:59]: Applied

File: site/ko/blog/new-in-devtools-108/index.md [hunk 2]
```diff
@@ -26,38 +27,41 @@ draft: true
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/NJVIK95TtKaXxzNVoGI6.png", alt="Recorder 패널에서의 XPath와 텍스트 셀렉터.", width="800", height="579" %}
 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/7441acfff5d9dfd373742797d2db46a809c9df67 #}
 
 Chromium issues: [1327206](https://crbug.com/1327206),[1327209] (https://crbug.com/1327209)
 
 
 <!-- ## Step through comma-separated expressions {: #debugging } -->
-
+## 쉼표로 구분된 표현을 통한 {: #debugging }
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-108/index.md:56]: Applied

**Review by jecfish — APPROVED** [review]

File: site/ko/_partials/devtools/whats-new.md [hunk 3]
```diff
@@ -6,12 +6,12 @@
 
 {# ### Chrome 108 {: #chrome108 }
```
Comment (jecfish) [inline site/ko/_partials/devtools/whats-new.md:7]:
```suggestion
### Chrome 108 {: #chrome108 }
```

**Pull request merged**